cmake/setup_install.cmake

```
@@ -0,0 +1,125 @@
+# This file is part of 4C multiphysics licensed under the
+# GNU Lesser General Public License v3.0 or later.
+#
+# See the LICENSE.md file in the top-level for license information.
+#
+# SPDX-License-Identifier: LGPL-3.0-or-later
+
+function(four_c_add_dependency_to_settings _package_name)
```

```suggestion
function(_add_dependency_to_settings _package_name)
```

Let's stick to the convention to mark functions that we only use locally with a leading underscore.